pull-request/README.md

```diff
@@ -0,0 +1,94 @@
+# Creating a Pull Request
+This tutorial will walk you through how to request that someone else pulls your changes to their code into their project.
+This is called a **pull request**, usually abbreviated to PR.
+It is assumed that you are comfortable with using the fundamental git commands (`git add`, `git commit`, etc.) at a command prompt, and that you are familiar with pushing and pulling repositories to and from GitHub.
```

Comment:
Around here, perhaps it would be worthwhile to link to a resource that describes how to use the basics of git that would be appropriate for someone completely new to it.  We could also support people in real time through Matrix or wherever who are new to it also.